File: backend/components/metiers/administration_publique/utils/logger/samples/sample_logger_helper.test.py
# Test du helper logger d’exemple (Python, clé en main)
from .sample_logger_helper import sample_logger_helper
def test_sample_logger_helper(capsys):
    assert sample_logger_helper('clé en main') is True
    captured = capsys.readouterr()
    assert '[LOGGER SAMPLE] Exemple:' in captured.out

File: backend/components/metiers/administration_publique/utils/logger/samples/sample_logger_helper.test.js
// Test du helper logger d’exemple (clé en main)
const { sampleLoggerHelper } = require('./sample_logger_helper');
describe('sampleLoggerHelper', () => {
  it('should log and return true', () => {
    expect(sampleLoggerHelper('clé en main')).toBe(true);
  });
});
